data/data/bootstrap/baremetal/files/etc/containers/systemd/image-customization.container

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=Customized Machine OS Image Server
+BindsTo=ironic-volume.service
+Requires=build-ironic-env.service extract-machine-os.service
+PartOf=ironic.service
+Wants=network-online.target
+After=network-online.target ironic-volume.service build-ironic-env.service extract-machine-os.service
+
+[Container]
```

Comment:
Curious: is it some openshift or podman feature? I cannot find any references to the [Container] section.